experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -156,15 +158,16 @@ mutable struct ProjectiveGlueing{
       # idQV = compose(g, f)
       # all(t->(pullback(idQV)(t) == t), gens(SQV)) || error("composition of maps is not the identity")
     end
-    return new{GlueingType, IsoType, IncType}(G, incP, incQ, f, g)
+    return new{GlueingType, IsoType1, IncType1, IsoType2, IncType2}(G, incP, incQ, f, g)
   end
 end
 
 ### type getters
-
+#=
+TODO: Do we need these?
```

Comment:
These did not work for me when I allowed more specs. 
Do we need them? Should we discard or repair them?

Reply:
Leave them as comments so that people can repair/extend them if needed.